posts/mathlib-changelog.md

```diff
@@ -0,0 +1,50 @@
+---
+author: 'David Chanin'
+category: 'Community projects'
+date: 2022-07-27 09:35:23 UTC+02:00
+description: ''
+has_math: false
+link: ''
+slug: mathlib-changelog
+tags: ''
+title: Introducing Mathlib Changelog
+type: text
+---
+
+Tldr; check out [mathlib-changelog.org](https://mathlib-changelog.org) to explore the historical changes to Mathlib, and find out what happened to that lemma you were using.
+
+
```

Comment:
```suggestion
<!-- TEASER_END -->

```
This will add the "Read more..." break on the index page. Feel free to move it if you think there's a more natural break point!